src/index.js

```diff
@@ -112,7 +112,7 @@ var DropinError = require('./lib/dropin-error');
 var Promise = require('./lib/promise');
 var wrapPromise = require('@braintree/wrap-promise');
 
-var VERSION = process.env.npm_package_version;
+var VERSION = require('../package.json').version;
```

Comment:
Is this required for your change? Or can we defer this change till after this PR is merged in?

Reply:
there was definitely some weirdness in the gulp function to build npm source (passing between streams and buffers iirc) that would have taken me a LOT of time to debug. After all the investigation I did, literally the only places in drop-in source code I found, it was being set in javascript and consumed in javascript. this PR is draft for a reason, and i still have one more piece of source code i wanna check before merging, but i'm 95% certain it can all go.


(i hope i don't regret this comment 😂 )

Reply:
For posterity's sake: it was set up this way specifically so post-build the actual string of the version would be written to the files instead of a reference to the package. I'm adding another line of gulp-replace as it supports streams unlike gulp-envify.

Reply:
(further, there's a _lot_ of version replacement that isn't working properly on master, will tackle in a future PR)

Reply:
More context on this?

Reply:
https://github.com/braintree/braintree-web-drop-in/pull/505